src/cron/schedule-google-calendar.js

```diff
@@ -34,7 +36,10 @@ const scheduleEventNotification = async ({ client, event }) => {
     );
   }
 
-  const cronExpression = dateToCronExpression(event.start.dateTime);
+  const startDate = new Date(event.start.dateTime);
+  startDate.setTime(startDate.getTime() - minutesBeforeEvent);
```

Comment:
I think this way would also work, so we avoid creating the minutes in milliseconds to do the transformation. What do you think?

![image](https://github.com/user-attachments/assets/fe8e3098-57ad-4680-a905-7bca213677cb)

```suggestion
  const startDate = new Date(event.start.dateTime);
  startDate.setMinutes(startDate.getMinutes() - 10);
```

We could still have a variable called minutesBeforeEvent but it would contain `10` instead of milliseconds

Reply:
Work for me, thank you!